website/docs/enterprise/getting-started/install-enterprise-cli.mdx

```diff
@@ -71,6 +70,9 @@ Please use the following command to start the installation wizard of Weave GitOp
        --version="0.35.0" \
        --domain-type="localhost" \
        --password="admin123"
+       --repo-url="ssh://git@github.com/my-org-name/my-repo-name"
+       --branch="main"
+       --repo-path="clusters/my-cluster"
```

Comment:
have you tested whether you could bootstrap with this command?

Reply:
yes, I've tested it with the same command and everything went well
![image](https://github.com/weaveworks/weave-gitops/assets/10185327/0202b655-719d-4261-9c84-57c0e7eb317a)

Reply:
cool, i guess i meant before without the `\` that i dont think it would work. 

```
➜✗  gitops bootstrap \                                                                            
    --kubeconfig=$HOME/.kube/config \
    --private-key=$HOME/.ssh/id_rsa \
    --private-key-password="" \
    --version="0.35.0" \
    --domain-type="localhost" \
    --password="admin123" \
    --repo-url="ssh://git@github.com/my-org-name/my-repo-name" \
    --branch="main" \
    --repo-path="clusters/my-cluster"
► creating client to cluster
✔ created client to cluster: https://127.0.0.1:52179
◎ checking flux
► verifying flux installation
✔ flux is installed

```